atlasdb-client-protobufs/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.6'
```

Comment:
Did you regenerate the protobufs. All the `*Persisitence.java` files should be regenerated and changes, if any be merged.